src/main/java/reposense/parser/RepoConfigCsvParser.java

```diff
@@ -41,8 +43,19 @@ protected void processLine(List<RepoConfiguration> results, String[] elements) t
         String location = getValueInElement(elements, LOCATION_POSITION);
         String branch = getValueInElement(elements, BRANCH_POSITION);
         List<String> ignoreGlobList = getManyValueInElement(elements, IGNORE_GLOB_LIST_POSITION);
+        String ignoreStandaloneConfig = getValueInElement(elements, IGNORE_STANDALONE_CONFIG_POSITION);
 
-        RepoConfiguration config = new RepoConfiguration(location, branch, ignoreGlobList);
+        if (!ignoreStandaloneConfig.equalsIgnoreCase(IGNORE_STANDALONE_CONFIG_KEYWORD)
+                && !ignoreStandaloneConfig.isEmpty()) {
+            logger.warning(
+                    "Ignoring unknown value " + ignoreStandaloneConfig + " used to ignore standalone config.");
```

Comment:
"Ignoring unknown value " + ignoreStandaloneConfig + " **in** ignore standalone config **column**.", since it technically isn't being used.